com.unity.netcode.gameobjects/Runtime/Transports/UNET/UNetTransport.cs

```diff
@@ -281,6 +281,26 @@ private ConnectionConfig GetConfig()
 
             return connectionConfig;
         }
+
```

Comment:
I'm not a massive fan of having these seperate functions I would have preferent to just modify the Port variables themselves to have get/set functions which would access the override but then there didnt appear to still allow them to exist inside the inspector. If there is a way and we prefer that idea then let me know and I can update